src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -273,6 +280,9 @@ DECLARE_API(IP2MD)
         return Status;
     }
     
+    ExtOut("Test output for IP2MD %16x\n", IP);
+    return Status;
```

Comment:
The command isn't complete finished or working yet. I'll put a comment that this is temporary.